setup.py

```diff
@@ -13,7 +13,7 @@
 import os
 import sys
 import warnings
-
+import numpy as np
```

Comment:
That's really wired, please don't use numpy in `setup.py` (by this reason all CI broken)

Reply:
ok removed this.  Didn't realize that you could include the numpy header files for the cython code without importing numpy directly.